### PR TITLE
chore: remove swatches and views from types

### PR DESCRIPTION
**Pull request by luqven**
This PR removes the `IProductImageData` type, since it's a duplicate of the `IImgixCustomAttribute` type. It also removes references to `swatches` and `size_type` since those are out of scope for this MVP.

### CODE REVIEW

**Comment by luqven** [comment]
Current dependencies on/for this PR:
* next
  * **PR #256** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/256" target="_blank">Graphite</a> 
    * **PR #258** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/258" target="_blank">Graphite</a> 
      * **PR #259** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/259" target="_blank">Graphite</a> 
        * **PR #257** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/257" target="_blank">Graphite</a> 
          * **PR #267** <a href="https://app.graphite.dev/github/pr/imgix/sf-commerce-cloud/267" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: frontend/src/components/ProductPageImages.tsx [hunk 1]
```diff
@@ -33,11 +33,11 @@ export const ProductPageImages = ({
       <>
         {/* we have to create a fragment around jsx elements otherwise
         typescript will throw an error */}
-        {images?.map((image: IProductImage) => {
+        {images?.map((image: IImgixCustomAttributeImage) => {
           return (
             <ProductImageContainer
               onOpenBreakoutClick={onClick}
-              image={{ ...image } as IProductImage}
+              image={{ ...image } as IImgixCustomAttributeImage}
```
Comment (frederickfogerty) [inline frontend/src/components/ProductPageImages.tsx:40]: This... shouldn't be necessary

**Review by frederickfogerty — APPROVED** [review]
LGTM. There seem to be a lot of `as ___` being used in TS - beware that this can lead to subtle bugs. It's better not to use these if possible

File: frontend/src/components/ProductPageImages.tsx (hunk 1 above)
Comment (luqven) [reply frontend/src/components/ProductPageImages.tsx:40]: oops-did-i-do-that

Left over from when the re-exported type had a different name from the type the component expected. Should be able to remove this.

**Comment by luqven** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).


Job Reference: 6cwJaETgMG17vfBY88e2

**Pull request merged**